ledger/participant-integration-api/src/test/lib/scala/platform/store/backend/PruningDtoQueries.scala

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.store.backend
+
+import java.sql.Connection
+
+import anorm.RowParser
+import anorm.SqlParser.{long, str}
+import anorm.~
+import com.daml.platform.store.backend.common.ComposableQuery.SqlStringInterpolation
+import com.daml.platform.store.backend.common.SimpleSqlAsVectorOf._
+
+object PruningDto {
+
+  case class Create(seqId: Long)
+  case class Consuming(seqId: Long)
+  case class NonConsuming(seqId: Long)
+  case class Divulgence(seqId: Long)
+
+  case class CreateFilterStakeholder(seqId: Long, party: Long)
```

Comment:
Maybe invert and have `Filter_[Create/Consuming/Nonconsuming]_[Stakeholder/NonStakeholder]`. Usually it's easier to explore and use prefix tree -like structures. The same for above (maybe prepend `Event`)

Reply:
Do you think I can introduce underscores in the class/method/parameter names?

What do you think about that, with the approach I coded currently, we can group like this:
```
create
createFilterStakeholder
createFilterNonStakeholder

consuming
consumingFilterStakeholder
consumingFilterNonStakeholder

nonConsuming
nonConsumingFilter
```
?

Reply:
> Do you think I can introduce underscores in the class/method/parameter names?

Maybe not warranted here since I see the names are rather short. Wrote it like that for making my point more readable. 

> What do you think about that, with the approach I coded currently, we can group like this:

Yes, but autocomplete doesn't work like that and likes prefix trees better ;) just nitpicking, leave it as you wish

Reply:
Made `event` and `filter` the prefixes